### PR TITLE
Add new official logos for pygame - Community Edition

### PR DESCRIPTION
**Pull request by Mega-JC**
Upon request by some contributors, other community members (#2234) and in order to have a better distinction between [upstream pygame](https://github.com/pygame/pygame) and pygame-ce, @kadir014 and I came up with a new logo for the pygame-ce project. The first concepts were created mid-October 2023, eventually leading to the current set of changes in this PR.

I've stayed conservative with stylistic changes to the pygame snake to preserve familiarity. The pupils are only slightly larger and positioned in a more friendly way.

Logo  |  LoFi | Powered 
:-------------------------:|:-------------------------:|:-------------------------:
image | image | image

Docs: Light Theme  |  Docs: Dark Theme
:-------------------------:|:-------------------------:
image  |  image
image  |  image

### Approximate README.rst preview
image

### CODE REVIEW

**Comment by damusss** [comment]
I love this new logo :)

**Comment by novialriptide** [comment]
I think the "ce" part of the logo should be smaller and should pronounce "community edition".

**Comment by Starbuck5** [comment]
> I think the "ce" part of the logo should be smaller and should pronounce "community edition".

I'd rather leave it as "ce".

"community edition" is a lot of text for a logo, even as banner-like as it is.

File: docs/reST/logos.rst [hunk 1]
```diff
@@ -1,47 +1,57 @@
...
 
-There is a higher resolution layered photoshop image
-available `here <https://www.pygame.org/ftp/pygame_logo.psd>`_. *(1.3 MB)*
 
-Legacy logos
-------------
+Upstream & legacy logos
+-----------------------
 
 .. container:: fullwidth
```
Comment (Starbuck5) [inline docs/reST/logos.rst:43]: I get a warning about this from Sphinx.

`C:\Users\charl\Desktop\pygame-ce\docs\reST\logos.rst:43: ERROR: Content block expected for the "container" directive; none found.`

Comment (Mega-JC) [reply docs/reST/logos.rst]: Fixed.

**Review by Starbuck5 — APPROVED** [review]
Thank you for pushing this forward Mega, and thanks to Lucy for the original design concept.

I like these new logos.

**Review by gresm — CHANGES_REQUESTED** [review]
Please update 
https://github.com/pygame-community/pygame-ce/blob/6f7aeea6e4ef9f9fd6f67e56bf63ea77e9a1d961/src_py/meson.build#L30-L33 
```meson
data_files = files( 
    'freesansbold.ttf', 
    'pygame_icon.bmp', 
    'pygame_ce_logo.png',
    'pygame_ce_powered.png',
) 
```
(and include copies of bundled images in ``src_py/`` - likely moving those files to a different folder will be needed in the future), so the code could access these images:
```py
import pygame

pygame.init()
powered_splash = pygame.image.load(pygame.pkgdata.getResource("pygame_ce_powered.png"))
```

Otherwise LGTM :+1:

**Comment by Mega-JC** [comment]
The sole goal behind this PR was to replace the previous logos by providing drop-in replacements, while making file name and link adjustments at all the places necessary.

Anything else feels out of scope.

**Review by MyreMylar — APPROVED** [review]
LGTM, well done to everyone who worked on these. Let's get them in I say 👍 

I do think we should consider - at some point - having a second style of logo that is more subdued and 'serious' looking. This is just because videogames is a broad category and while many have the same cheery, fun colourful ambience of the traditional pygame style  there are also plenty that are sadgames in a grim future of only war where another style of logo might blend in more easily.

Plenty of precedent for long running media organisations to have many logos, and logo variants.

As an example of something we could try adding as a variant that might suit many users - a monochrome variant, something that can be easily colourized to fit an app/game's colour scheme:

pygame_ce_monochrome

Colourized example:
pygame_ce_colorized_example

**Comment by MyreMylar** [comment]
> Please update
> 
> https://github.com/pygame-community/pygame-ce/blob/6f7aeea6e4ef9f9fd6f67e56bf63ea77e9a1d961/src_py/meson.build#L30-L33
> 
> ```meson
> data_files = files( 
>     'freesansbold.ttf', 
>     'pygame_icon.bmp', 
>     'pygame_ce_logo.png',
>     'pygame_ce_powered.png',
> ) 
> ```
> 
> (and include copies of bundled images in `src_py/` - likely moving those files to a different folder will be needed in the future), so the code could access these images:
> 
> ```python
> import pygame
> 
> pygame.init()
> powered_splash = pygame.image.load(pygame.pkgdata.getResource("pygame_ce_powered.png"))
> ```
> 
> Otherwise LGTM 👍

Let's move this idea to a new issue/PR so we can resolve the long-running logo switchover issue.

**Pull request merged**